content/docs/latest/installing/vms/hyper-v.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Running Flatcar Container Linux on Hyper-V
+title: Running on Hyper-V
+weight: 30
+aliases:
+    - ../../os/booting-on-hyper-v
+    - ../../cloud-providers/booting-on-hyper-v
+---
+
+_While we always welcome community contributions and fixes, please note that Hyper-V is not an officially supported platform at this time. (See the [platform overview](/#installing-flatcar).)_
+
+These instructions will walk you through running Flatcar Container Linux on Hyper-V.
+
+## Choose a channel
+
+Flatcar Container Linux is designed to be updated automatically with different schedules per channel. You can [disable this feature][update-strategies], although we don't recommend it. Read the [release notes][release-notes] for specific features and bug fixes.
+
+## Creating a config-drive
+
+Cloud-config can be specified by attaching a [config-drive](https://github.com/flatcar/coreos-cloudinit/blob/master/Documentation/config-drive.md) with the label `config-2`. This is commonly done through whatever interface allows for attaching CD-ROMs or new drives.
```

Comment:
Does the config drive also support ignition? Then I would rather talk about userdata in general.

Reply:
The config drive is basically the same as the openstack workflow, and it is implementend by the coreos-cloudinit. So it is not implemented by Ignition, it's just a fallback service.

Reply:
Ok, then I would say something like
```suggestion
While Ignition is not supported on a config-drive, you can provide a Cloud-config following the coreos-cloudinit subset.
The cloud-config can be specified by attaching a [config-drive](https://github.com/flatcar/coreos-cloudinit/blob/master/Documentation/config-drive.md) with the label `config-2`. This is commonly done through whatever interface allows for attaching CD-ROMs or new drives.
```